docs/en/styling-and-theming/basic-usage.md

```diff
@@ -116,68 +113,64 @@ export default {
 
 ## Specifying a default application theme
 
--   Using the theme injector function, [`registerThemeInjector()`](./supplemental.md#making-themeable-applications), to automatically provide a custom theme to all themeable widgets in an application
+The `theme` middleware can be used to set the application theme. To set a "default" or initial theme the `theme.set` function can be used with the `theme.get` function to determine if the theme needs to be set. Setting the default theme should be done in the application's top level widget.
 
-> src/main.tsx
+> src/App.tsx
 
 ```tsx
-import renderer from '@dojo/framework/core/vdom';
-import { tsx } from '@dojo/framework/core/vdom';
-import Registry from '@dojo/framework/core/Registry';
-import { registerThemeInjector } from '@dojo/framework/core/mixins/Themed';
+import { create, tsx } from '@dojo/framework/core/vdom';
+import theme from '@dojo/framework/core/middleware/theme';
 
-import myTheme from './themes/MyTheme/theme';
-import App from './widgets/App';
+import myTheme from '../themes/MyTheme/theme';
 
-const registry = new Registry();
-registerThemeInjector(myTheme, registry);
+const factory = create({ theme });
 
-const r = renderer(() => <App />);
-r.mount({ registry });
+export default factory(function App({ middleware: { theme }}) {
+	// if the theme isn't set, set the default theme
+	if (!theme.get()) {
+		theme.set(myTheme);
+	}
+	return (
+		// you application's widgets
```

Comment:
your*, although we may want to remove that word. I've tried to avoid 'you'/'your' in these guides so they don't read like the users are being talked at, rather talked with